x/hard/legacy/v0_11/types.go

```diff
@@ -18,12 +18,6 @@ const (
 	// ModuleName name that will be used throughout the module
 	ModuleName = "harvest"
 
-	// LPAccount LP distribution module account
-	LPAccount = "harvest_lp_distribution"
-
-	// DelegatorAccount delegator distribution module account
-	DelegatorAccount = "harvest_delegator_distribution"
-
```

Comment:
these shouldn't be removed. Anything in `legacy/` for old versions is meant to be a copy of the types from the old version, so shouldn't be modified

Reply:
Addressed in https://github.com/Kava-Labs/kava/pull/796/commits/ce1d67ed48570c482c85f4ec2695f4c8d96607af